README.md

```diff
@@ -17,9 +17,8 @@ import 'clipboard-copy-element'
 ```html
 <clipboard-copy
       for="blob-path"
-      class="btn btn-sm BtnGroup-item tooltipped tooltipped-s"
-      aria-label="Copy file path to clipboard"
-      copied-label="Copied!">
+      class="btn btn-sm BtnGroup-item"
+      title="Copy file path to clipboard">
```

Comment:
This `title` is still not needed, isn't it? The button text "Copy path" should be enough.